pombola/south_africa/data/members-interests/convert_to_import_json.py

```diff
@@ -417,25 +537,32 @@ def mp_to_person_slug(self, mp):
             return person.slug
         except Person.DoesNotExist:
             try:
-                name_parts = re.findall(r'(.*?), (.*)', muddled_name)
-                person = Person.objects.get(Q(slug__contains=slugify(name_parts[0][0])) & Q(slug__contains=slugify(name_parts[0][1])))
-                return person.slug
+                name_base = re.findall(r'(.*?), (.*)', muddled_name.replace('-', ','))
+                if name_base:
+                    name_parts = name_base[0]
+                    try:
+                        person = Person.objects.get(Q(slug__contains=slugify(name_parts[0])) & Q(slug__contains=slugify(name_parts[1])))
+                        return person.slug
+                    except Person.MultipleObjectsReturned:
+                        print('Multiple people matched', name_parts, slug if slug else '')
```

Comment:
perhaps explicitly return None here - that's the behaviour we're relying on

Reply:
Done.